Fixed problem starting tsserver

svenefftinge opened the pull request.

prabirshrestha commented:
👍 verified it works on windows.

prabirshrestha commented on src/lsp-server.ts:
```diff
@@ -50,8 +50,8 @@ export class LspServer {
-        const path = findPathToModule(this.rootPath(), 'typescript/bin/tsserver')
-        return path || 'tsserver'
+        let path = findPathToModule(this.rootPath(), '.bin/tsserver') || 'tsserver'
+        return isWindows() ? path + ".cmd" : path
```
I still think it might be better to refactor this into a function since cli help also needs this logic.

```typescript
.option('--tsserver-path <path>',
        `Specifiy absolute path to tsserver. example: --tsserver-path=${isWindows() ? 'c:\\tsc\\tsserver.cmd' : '/bin/tsserver'}`,
        isWindows() ? 'tsserver.cmd' : 'tsserver')
```

angelozerr commented:
Works like a charm with my Windows too.

svenefftinge commented:
Improved resolution of tsserver executable (fixes #2)
Added lookup precedence to be 
1) workspace relative
2) global installation
3) shipped with language server

Fixed problem with windows
